.ci/scripts/check_c10_sync.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
where is this file being used?

Reply:
whoops! supposed to be the thing that the workflow calls